docs/general-information/general-information.md

```diff
@@ -12,12 +12,12 @@ anchors:
 ---
 Overview
 --------
-BELLATRIX is not a single thing it contains multiple framework libraries, extensions and tools. The tool is built to be cross-platform.
+BELLATRIX is not a single thing, it contains multiple framework libraries, extensions and tools. The tool is built to be cross-platform.
 
 Simple Installation
 ------------------
-1. Download the BELLATRIX
-2. Unzip it. Open BellatrixTestFramework.sln in Visual Studio 
+1. Download BELLATRIX.
```

Comment:
Download the BELLATRIX file/zip.

Reply:
I'll wait for all the feedback before making changes.